core/src/main/res/values/strings.xml

```diff
@@ -262,7 +262,6 @@
     <string name="download_plugin_label">Download Plugin</string>
     <string name="no_playback_plugin_title">Plugin Not Installed</string>
     <string name="no_playback_plugin_or_sonic_msg">For variable speed playback to work, you have to install a third party library or enable the experimental Sonic player [Android 4.1+].\n\nTap \'Download Plugin\' to download a free plugin from the Play Store.\n\nAny problems found using this plugin are not the responsibility of AntennaPod and should be reported to the plugin owner.</string>
-    <string name="set_playback_speed_label">Playback Speeds</string>
```

Comment:
Nope, we only ever edit strings.xml. Else we would get in conflict with Transifex.